Apps/WebClient/src/ClientApp/src/components/timeline/entryCard/entrycard.vue

```diff
@@ -26,6 +28,17 @@ export default class EntrycardTimelineComponent extends Vue {
     @Prop() iconClass!: string;
     @Prop({ default: true }) allowComment!: boolean;
     @Prop({ default: true }) showDetailsButton!: boolean;
+    @Prop() viewDetails!: boolean;
```

Comment:
instead of specifying this property in all the other entries, you can just set the default value without having to define it everywhere

Reply:
Also, isMobileDetails makes it much more readable